docs/content/docs/dev/table/sql/create.md

```diff
@@ -557,6 +558,59 @@ INSERT INTO my_ctas_table SELECT id, name, age FROM source_table WHERE mod(id, 1
 
 {{< top >}}
 
+## [CREATE OR] REPLACE TABLE
+```sql
+[CREATE OR] REPLACE TABLE [catalog_name.][db_name.]table_name
+[COMMENT table_comment]
+WITH (key1=val1, key2=val2, ...)
+AS select_query
+```
+
+**Note** RTAS has the following semantic:
+* REPLACE TABLE AS SELECT statement: the target table to be replaced must exist, otherwise, an exception will be thrown.
+* CREATE OR REPLACE TABLE AS SELECT statement: the target table to be replaced will be created if it does not exist; if it does exist, it'll be replaced.
+
+Tables can also be replaced(or created) and populated by the results of a query in one [CREATE OR] REPLACE TABLE(RTAS) statement.  RTAS is the simplest and fastest way to replace and insert data into a table with a single command.
```

Comment:
nit: I'd like to remove `also`, for there's no context in here. It's not like ctas part.'
```suggestion
Tables can be replaced(or created) and populated by the results of a query in one [CREATE OR] REPLACE TABLE(RTAS) statement.  RTAS is the simplest and fastest way to replace and insert data into a table with a single command.
```